core/src/main/java/me/leoko/advancedban/Universal.java

```diff
@@ -142,6 +142,14 @@ public boolean isBungee() {
         return mi.isBungee();
```

Comment:
Since MethodInterface#isBungee is deprecated, Universal#isBungee should also be so